src/app/content/studyGuides/reducer.spec.ts

```diff
@@ -1,15 +1,19 @@
-import { HighlightsSummary } from '@openstax/highlighter/dist/api';
 import { receiveFeatureFlags } from '../../actions';
 import { studyGuidesFeatureFlag } from '../constants';
+import { SummaryHighlights } from '../types';
 import * as actions from './actions';
 import reducer, { initialState } from './reducer';
 
 describe('study guides reducer', () => {
   it('receive study guides', () => {
-    const summary = { asd: 'asd' } as any as HighlightsSummary;
-    const state = reducer(undefined, actions.receiveStudyGuides(summary));
+    const summary = {
+      location: {
+        page: [],
+      },
+    } as SummaryHighlights;
```

Comment:
if this is about studyguides, `Highlights` shouldn't be mentioned anywhere, it's a bit confusing

Reply:
I've treated `StudyGuides` as special kind of `Highlights` since in the end, they still use things like
`highlightClient.getHighlights` 
`GetHighlightsSetsEnum`
`GetHighlightsSourceTypeEnum`
`GetHighlightsColorsEnum`

Reply:
@janKozie1 i think that is a good frame of mind for this stuff